src/sdk/pynni/nni/README.md

```diff
@@ -46,6 +46,9 @@ Note that SMAC only supports a subset of the types in [search space spec](../../
 
 Batch allows users to simply provide several configurations (i.e., choices of hyper-parameters) for their trial code. After finishing all the configurations, the experiment is done.
 
+**Gridsearch**

```

Comment:
please make this part more detail, for example, what search space spec it supports and the behaviors of the supported spec.

Reply:
Have added